tests/phpunit/api/v3/ContributionTest.php

```diff
@@ -911,32 +936,14 @@ public function testCreateContributionSourceInvalidContact() {
     $this->callAPIFailure('contribution', 'create', $params, 'contact_id is not valid : 999');
   }
 
-  public function testCreateContributionSourceInvalidContContact() {
-
-    $params = [
-      'contribution_contact_id' => 999,
```

Comment:
We have adequate cover for using an invalid contact id in syntax conformance - in general these failure tests have been rationalised from the early days